nimble/host/include/host/ble_iso.h

```diff
@@ -19,29 +19,112 @@
 
 #ifndef H_BLE_ISO_
 #define H_BLE_ISO_
+
+/**
+ * @file ble_iso.h
+ *
+ * @brief Bluetooth ISO
+ * @defgroup bt_iso Bluetooth ISO
+ * @ingroup bt_host
+ * @{
+ */
+
 #include "syscfg/syscfg.h"
 
+/**
+ * @defgroup ble_iso_events ISO Events
+ * @{
+ */
+
 /** ISO event: BIG Create Completed */
 #define BLE_ISO_EVENT_BIG_CREATE_COMPLETE                  0
 
 /** ISO event: BIG Terminate Completed */
 #define BLE_ISO_EVENT_BIG_TERMINATE_COMPLETE               1
 
+/** @} */
+
 #include <inttypes.h>
 
+/** Broadcast Isochronous Group (BIG) descriptor */
 struct ble_iso_big_desc
 {
+    /**
+     * The identifier of the BIG. Assigned by the Host when a new BIG is
+     * created. The value shall be between 0x00 and 0xEF.
+     */
     uint8_t big_handle;
+
+    /**
+     * The maximum time in microseconds for transmission of PDUs of all BISes in
+     * a BIG event. The value of BIG_Sync_Delay shall equal the time from the
+     * anchor point to the BIG Synchronization point and shall be between
+     * 0x0000EA and 0x7FFFFF.
+     */
```

Comment:
Ditto, probably just a short description like below would be sufficient 
```suggestion
    /**
     * The maximum time in microseconds for transmission of PDUs of all BISes in
     * a BIG event.
     */
```